parts/k8s/kubernetescustomscript.sh

```diff
@@ -486,6 +486,10 @@ configACIConnectorAddon() {
     sed -i "s|<kubernetesACIConnectorKey>|$(echo $ACI_CONNECTOR_KEY)|g" "/etc/kubernetes/addons/aci-connector-deployment.yaml"
 }
 
+configAADPodIdentityAddon() {
+    sed -i "s|<kubernetesAADPodIdentityCredentials>|$AAD_POD_IDENTITY|g" "/etc/kubernetes/addons/aad-pod-identity-deployment.yaml"
```

Comment:
It looks like we aren't actually passing in `AAD_POD_IDENTITY` or `AAD_POD_IDENTITY_ADDON` env vars yet. May I suggest that we drop the "boolean" `AAD_POD_IDENTITY_ADDON` var, and simply check for non-empty string value of `AAD_POD_IDENTITY` to determine whether or not we want to do the sed replacement?

Reply:
Thank you, Jack. Since we don't have anything to pass and use values from azure.json, this is not required. So have removed this.